### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by itfdw** [alexfu/form-validator#1]
[
![screenshot_1](https://cloud.githubusercontent.com/assets/20770526/18636367/6e28e362-7e80-11e6-89e6-e2d9c505bf04.png)
](url)

**Comment by alexfu** [comment]
@itfdw do you have jitpack included as one of your dependency repos?

```
buildscript {
  repositories {
    maven { url 'https://jitpack.io' }
  }
}
```

**Comment by itfdw** [comment]
yes!

**Comment by alexfu** [comment]
@itfdw I can't reproduce so you'll need to provide more info. what does your `build.gradle` file look like? (both of them).

**Comment by alexfu** [comment]
@itfdw I just hit this and it's because the documentation in the README is incorrect. Apologies. Take a look at this commit: https://github.com/alexfu/form-validator/commit/e9618b14561b756956886dcbdebef0dbca6584cb

Fixed in e9618b14561b756956886dcbdebef0dbca6584cb.
